cmd/application.go

```diff
@@ -46,18 +46,34 @@ var applicationCmd = &cobra.Command{
 var applicationCreateCmd = &cobra.Command{
 	Use:   "create",
 	Short: "Create an application",
-	Long:  "Create an application",
+	Long: `Create an application.
+If no app name is passed, a default app name will be generated as per $dir-$random-4-char-str or if prefix configured, $prefix-$random-4-char-str
```

Comment:
Seems a bit too long, even when doing `odo create --help` this seems long.

Can you shorten it to just: `If no app name is passed, a default app name will be auto-generated` 

There's no need to tell the user what format it'll be, similar to how OpenShift or `kubectl` doesn't

Reply:
Ack